src/chrome/content/rules/SpringerNature.com.xml

```diff
@@ -0,0 +1,59 @@
+<!--
+	Refused:
+		authors.springernature.com
+		compliance.springernature.com
+		helpdesk.springernature.com
+		unifiedconcur.springernature.com
+
+	No working URL known:
+		citrix-nz.springernature.com
+		eas.springernature.com
+		ews.springernature.com
+		idp.springernature.com
+		media.springernature.com
+		owa.springernature.com
+
+	Invalid certificate:
+		click.springernature.com
+		editorial-jobs.springernature.com
+		www.search.springernature.com
+		sndigital.springernature.com
+
+	Mixed content:
+		group.springernature.com
```

Comment:
Is there a specific issue? I wasn't able to find one and the site defaults to https for me.

Reply:
If I don't specify a test url, it's usually on `$`. It seems indeed gone in that case.